example/lib/03.control/code_editor.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:flutter/material.dart';
+
+import 'constants/constants.dart';
+import 'widgets/code_box.dart';
+import 'widgets/code_editor_appbar.dart';
+
+class CodeEditor extends StatefulWidget {
+  final String language;
+  final String theme;
+  final bool reset;
```

Comment:
Android Studio would add a blank line here if you use its formatter. I am not sure if `dart format` adds it or Android Studio, but it's better to add one.